tests/helixpublishwitharcade.proj

```diff
@@ -29,57 +30,77 @@
 
     <WaitForWorkItemCompletion>true</WaitForWorkItemCompletion>
     <CoreRootDirectory>$(TestWorkingDir)\Tests\Core_Root</CoreRootDirectory>
-  </PropertyGroup>
-
-  <PropertyGroup Condition=" '$(TargetsWindows)' == 'true' ">
-    <HelixPythonPath>%HELIX_PYTHONPATH%</HelixPythonPath>
-    <RunnerScript>%HELIX_CORRELATION_PAYLOAD%\runtest_helix.py</RunnerScript>
-  </PropertyGroup>
-
-  <PropertyGroup Condition=" '$(TargetsWindows)' != 'true' ">
-    <HelixPythonPath>$HELIX_PYTHONPATH</HelixPythonPath>
-    <RunnerScript>$HELIX_CORRELATION_PAYLOAD/runtest_helix.py</RunnerScript>
+    <TestEnvFileExtension Condition=" '$(TargetsWindows)' == 'true' ">.cmd</TestEnvFileExtension>
+    <TestEnvFileExtension Condition=" '$(TargetsWindows)' != 'true' ">.sh</TestEnvFileExtension>
   </PropertyGroup>
 
   <ItemGroup>
     <HelixCorrelationPayload Include="$(CoreRootDirectory)" />
     <XUnitWrapperDlls Include="$(TestWorkingDir)\**\*.XUnitWrapper.dll" />
+    <Scenarios Include="$(_Scenarios.Split(','))" />
   </ItemGroup>
 
-  <Target Name="CopyRunnerScriptToCoreRootDirectory" BeforeTargets="SubmitTestsToHelix">
-    <Copy SourceFiles="$(MSBuildThisFileDirectory)\runtest_helix.py" DestinationFolder="$(CoreRootDirectory)" />
+  <Target Name="CreateTestEnvFiles">
+    <ItemGroup>
```

Comment:
Maybe comment that this will create one test env file per xunit-wrapper/scenario combination.

Reply:
Sounds good!